### PR TITLE
feat: add a talos blog post for openstack-flex

### PR DESCRIPTION
**Pull request by cloudnull**
This change adds a new blog post for Talos, which I'll be using for
content in other application centric posts being developed for GA.

1. Updates the CSS to have a better reading experience on long content.
2. Create a talos cluster
3. Create a storage cluster with longhorn
4. Create a crunch postgres cluster
5. Craete a cockroachDB cluster
6. Deploy MetalLB with, covering allowed address pairs
7. Create a highly available Teleport cluster

All blog posts work together in the series to show how administrators
can leveral openstack-flex.

Related-Issue: https://rackspace.atlassian.net/browse/OSPC-118

### CODE REVIEW

File: docs/overrides/stylesheets/adr.css
Comment (cloudnull) [inline docs/overrides/stylesheets/adr.css:1]: resolves CSS bug on the blog that made reading long content difficult.

Comment (cloudnull) [reply docs/overrides/stylesheets/adr.css:1]: image

File: docs/blog/posts/2024-11-04-running-talos-on-openstack-flex.md [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+
+In essence, Talos offers more by providing less—less complexity, less overhead, and fewer security concerns. This minimalistic yet powerful approach enhances security, efficiency, resiliency, and consistency. For developers working with OpenStack and specifically OpenStack Flex, Talos presents a compelling operating system choice that aligns perfectly with the goals of modern open infrastructure native applications.
+
+## Creating a cluster via the CLI on OpenStack
+
+In this guide, we will create an HA Kubernetes cluster in OpenStack with 3 worker node. We will assume an existing some familiarity with OpenStack. If you need more information on OpenStack specifics, please see the official OpenStack documentation.
+
+``` mirmaid
```
Comment (cloudnull) [inline docs/blog/posts/2024-11-04-running-talos-on-openstack-flex.md:31]: image

screenshot to show how the diagram will look

File: docs/blog/posts/2024-11-04-running-talos-on-openstack-flex.md
Comment (devx) [inline docs/blog/posts/2024-11-04-running-talos-on-openstack-flex.md:1]: +1 LGTM - Like the security focus and different networks not to mention the support for OVM.

File: docs/blog/posts/2024-11-05-running-metallb-on-openstack-flex.md
Comment (devx) [inline docs/blog/posts/2024-11-05-running-metallb-on-openstack-flex.md:1]: +1 LGTM - Very simple and straight way to add LB capabilities.

File: docs/blog/posts/2024-11-04-running-longhorn-on-openstack-flex.md
Comment (devx) [inline docs/blog/posts/2024-11-04-running-longhorn-on-openstack-flex.md:1]: I like the use of the talos patch as it helps people understand how you make changes to talos.

File: docs/blog/posts/2024-11-04-running-crunchy-postgres-on-openstack-flex.md
Comment (devx) [inline docs/blog/posts/2024-11-04-running-crunchy-postgres-on-openstack-flex.md:1]: +1 LGTM - I think in the future we should specify if it's usign OLMv0 or OLMv1.

File: docs/blog/posts/2024-11-04-running-cockroachdb-on-openstack-flex.md
Comment (devx) [inline docs/blog/posts/2024-11-04-running-cockroachdb-on-openstack-flex.md:1]: +1 LGTM - Same comment as the crunchy Postgres.

**Review by devx — APPROVED** [review]
+1  LGTM - Everything checkouts and very good manual documentation.

**Review by rackerchris — APPROVED** [review]
LGTM +1

**Pull request merged**